ouroboros-consensus-shelley-test/test/Main.hs

```diff
@@ -11,6 +11,10 @@ main = defaultMain tests
 
 tests :: TestTree
 tests =
+  testGroup "shelley"
+  [ Test.ThreadNet.RealTPraos.tests
+  ]
+  `asTypeOf`
```

Comment:
Forgot to remove?

Reply:
Gah, this is the second PR in which I've let that slip in :(